portal-web/test/functional/com/liferay/portalweb/macros/ItemSelector.macro

```diff
@@ -758,6 +758,58 @@ definition {
 		}
 	}
 
+	macro viewFilteredResultsViaAssetLists {
+		var key_assetType = "${assetType}";
+
+		if (isSet(assetListName)) {
+			Click(
+				key_title = "${assetListName}",
+				locator1 = "AssetLists#ASSET_LIST_ENTRY_TITLE");
+		}
+
+		Pause(locator1 = "3000");
```

Comment:
Hey @liuxingye I noticed in the last couple of pulls that you've been using a lot of hard pauses. Can we try to use wait functions (like assertElementPresent) instead?

Reply:
I refer to AssetListsAdmin.selectAsset to write the new macro, it also works without pause locally, but to be more stable, I added a pause finally.

Reply:
I checked my commits from last Tuesday, I used pause for 2 times. one is here, another is [ViewMentionViaBlogsEntryCommentAsGuest](https://github.com/austinchiang/liferay-portal/pull/986/files), use pause there to give enough time loading for blog things.